secant/Features/App/AppStore.swift

```diff
@@ -304,6 +311,37 @@ extension AppReducer {
                 state.route = .home
             }
 
+        case .deeplink(let url):
+            // get the latest synchronizer state
+//            var synchronizerStatus = WrappedSDKSynchronizerState.unknown
```

Comment:
Remove commented code if possible. If they are needed then let's open an issue and track this there